modules/javafx.graphics/src/main/java/com/sun/glass/ui/win/WinAccessible.java

```diff
@@ -887,8 +901,16 @@ private float[] get_BoundingRectangle() {
 
         Bounds bounds = (Bounds)getAttribute(BOUNDS);
```

Comment:
There is another instance where `(Bounds)getAttribute(BOUNDS)` is used in this File (in method `get_HorizontalViewSize()`)
Please check whether we need to apply scale there as well?

Reply:
That's a good question to ask. Both `get_HorizontalViewSize()` and `get_VerticalViewSize()` take the ratio of two bounds to compute a percentage between 0 and 100. Since the only use of the bounds in these methods is to divide two widths or heights, it doesn't matter whether or not the ratio is computed using unscaled or scaled values. Further, if I were to scale those values, I'd also need to scale the hard-coded `24` pixel estimate for the cell height. It seemed better to leave them alone.